src/jdk.jdi/share/classes/com/sun/jdi/ClassType.java

```diff
@@ -345,7 +345,7 @@ ObjectReference newInstance(ThreadReference thread, Method method,
                                           InvocationException;
 
     /**
-     * Returns a the single non-abstract {@link Method} visible from
+     * Returns the single non-abstract {@link Method} visible from
```

Comment:
I would think "Returns a single ..." because the implementation returns the first match it finds, from possibly many.

Reply:
I've accepted it, yet I'm still unsure whether ‘a‘ or ‘the’…